tests/dummy/app/app.js

```diff
@@ -4,7 +4,7 @@ import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
 // ember-browserify requires every package to be imported somewhere in the `app` tree
-import 'npm:common-tags';
+import 'common-tags';
```

Comment:
I've removed the line and the `ember-browserify` dependency and the tests are passing here. I'll wait the current build to finish and then I'll rebase.